app/src/main/java/org/greenstand/android/TreeTracker/viewmodels/NewTreeViewModel.kt

```diff
@@ -43,17 +40,10 @@ class NewTreeViewModel(
     var isNextButtonActive = !isDbhEnabled
         private set
 
-    val accuracyLiveData: LiveData<Int> = MutableLiveData<Int>().apply {
```

Comment:
Removing all references to accuracy data from the code. This is meaningless now that we rely only on convergence for all purposes in the app.